x/gov/migrations/v5/store.go

```diff
@@ -24,9 +24,6 @@ func MigrateStore(ctx sdk.Context, storeKey storetypes.StoreKey, cdc codec.Binar
 	params.ExpeditedMinDeposit = defaultParams.ExpeditedMinDeposit
 	params.ExpeditedVotingPeriod = defaultParams.ExpeditedVotingPeriod
 	params.ExpeditedThreshold = defaultParams.ExpeditedThreshold
-	params.BurnProposalDepositPrevote = defaultParams.BurnProposalDepositPrevote
```

Comment:
We've migrated in v4 (during gov params migration)

Reply:
See https://github.com/cosmos/cosmos-sdk/pull/15219#issuecomment-1449802058

Reply:
ah true, v5 is the next version not the current release one